profiles/views.py

```diff
@@ -99,7 +99,7 @@ def logout(request, next_page=None,
         next_page = request.POST.get(redirect_field_name,
                                      request.GET.get(redirect_field_name))
         # Security check -- don't allow redirection to a different host.
-        if not is_safe_url(url=next_page, host=request.get_host()):
+        if not is_safe_url(url=next_page, allowed_hosts=request.get_host()):
```

Comment:
what's going on here?

Reply:
This was a change in Django 2.1

`allowed_hosts is now a required argument of private API django.utils.http.is_safe_url().` and `The host parameter of django.utils.http.is_safe_url() is removed.`